geekodoc/tests/bad/procedure-informalfigure-in-step.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../rng/geekodoc5-flat.rnc" type="application/relax-ng-compact-syntax"?>
+<procedure xmlns="http://docbook.org/ns/docbook"
+    xmlns:xlink="http://www.w3.org/1999/xlink" version="5.1">
+    <title>An Procedure with a invalid step</title>
```

Comment:
A procedure

Reply:
as we are nitpicking already: it also neeeds to be "a_n_ invalid step"

Reply:
Resolved with cb2c3e0